x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -587,6 +587,9 @@ export class TaskRunner<
       ),
       schedule: resolveErr<IntervalSchedule | undefined, Error>(schedule, (error) => {
         if (isAlertSavedObjectNotFoundError(error, alertId)) {
+          this.logger.warn(
```

Comment:
WDYT about adding telemetry regarding this (in a separate ticket)? Do we have any insight into how often this happen?

Reply:
Yeah, we can create a separate issue for this. I think we would have to query the event log index for rule executions that end in an error status, but maybe would need a different field to aggregate on since the event log captures the `Saved object not found` message but since that contains the rule id, it's different for each rule. Currently it looks like the telemetry runs on the `.kibana` index.

Reply:
@chrisronline I opened a generic issue for adding information from the event log to telemetry: https://github.com/elastic/kibana/issues/101809